spec/iex/endpoints/ref_data_spec.rb

```diff
@@ -36,5 +36,13 @@
         end
       end
     end
+
+    context 'graciously handle parameter as string' do
+      subject { client.ref_data_isin('US0378331005') }
+
+      it 'converts ISIN from string to Array and do not errors out' do
+        expect { subject }.not_to raise_error
```

Comment:
This test doesn't test that the API actually works, aka that it returns the ISIN for the value you provide. Use an existing VCR that asks for 1 array item, aka we want 2 tests that it works the same way for  `"US0378331005"` and for `["US0378331005"]`.

Reply:
Also probably want to highlight in README both examples (you can pass 1 value or an array of values).

Reply:
Good point. Please check the [latest commit](https://github.com/dblock/iex-ruby-client/pull/70/commits/6c6a411277964cd4478c2b24324ae6d4415bbb7c)

Reply:
I still have an issue with `expect { subject }.not_to raise_error`, it's a test that says "make sure it doesn't throw an exception", which is weird. We expect all our calls not to throw errors ;)

Reply:
Also noticed that we have a nested block with different VCRs, the whole spec should look like this:

```ruby
describe '#ref_data_isin' do
   context 'with one asin', vcr: { cassette_name: 'ref-data/isin' } 
   context 'with multiple asins', vcr: ...
   context 'with invalid asins', vcr: ...
end
```

Reply:
@dblock done. What do you think now?